migration/identity-migration/src/main/java/io/camunda/migration/identity/MigrationHandler.java

```diff
@@ -10,20 +10,35 @@
 import io.camunda.service.exception.CamundaBrokerException;
 import io.camunda.zeebe.broker.client.api.BrokerRejectionException;
 import io.camunda.zeebe.protocol.record.RejectionType;
+import java.util.List;
 import java.util.concurrent.CompletionException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-public interface MigrationHandler {
+public abstract class MigrationHandler<T> {
+  static final int SIZE = 100;
+  protected final Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
:wrench: I'd rather have dedicated loggers for each handler, that makes it easier to figure out where logs come from.

Reply:
because the logger is created with getClass, each of MigrationHandler has an specific logger.